src/docs/plugins/plugins.mdx

```diff
@@ -128,6 +128,27 @@ or
 npm install styled-components
 ```
 
+## material-ui
+The `material-ui` plugin encapsulates `material-ui` styles into webcomponent's shadowRoot.
```

Comment:
could be more clear if we link material-ui, instead of using the same word twice and expecting them to know the difference :)
what do you think?
```suggestion
The `material-ui` plugin encapsulates [material-ui](https://material-ui.com/) styles into webcomponent's shadowRoot.
```

Reply:
Totally agreed !
I modified it, amended my previous commit and forced a push to keep git history clean.
Thank you for the feeback.